attic/rfc_metrics.md

```diff
@@ -36,14 +46,55 @@ Internally, the function would look the following:
 
 Example:
 ```r
-fn = function(truth, response, positive, ...) {
-  assert_binary(truth, response = response, positive = positive)
-  fn_cm(cm(truth, response, positive))
-}
+MeasureFairness = R6Class("MeasureFairness", inherit = Measure, cloneable = FALSE,
+  public = list(
+    fun = NULL,
+    na_value = NaN,
+    base_measure = NULL,
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function(name, measure) {
+      info = mlr3fairness::measures[[name]]
+      super$initialize(
+        id = paste0("fairness.", name),
+        range = c(info$lower, info$upper),
+        minimize = info$minimize,
+        predict_type = info$predict_type,
+        packages = "mlr3fairness",
+        man = paste0("mlr3::mlr_measures_fairness.", name)
```

Comment:
I think the `man` (this is where to find documentation does not make sense. 
I would set it to 'mlr_measures_fairness`

Also: Are you sure you need `mlr3::`?

Reply:
Emm, should we set to mlr3fairness:: or just ignore it.